CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 * Add Created At on Flaw Edit (`OSIDB-2945`)
 * Add Affect affectedness on Advanced Search (`OSIDB-2951`)
 
+### Changed
+* Remove Trackers section on Flaw Edit (`OSIDB-2954`)
+
```

Comment:
I think this should be added to a `unreleased` version instead of `2024.6.1` but to be honest I'm a bit lost with the changelog 😅

Reply:
Just updated.